test/functional/basic/gesture-e2e-specs.js

```diff
@@ -1,310 +1,363 @@
+            .move({ origin: el, x: 50, y: 0 })
+            .down()
+            .pause(100)
+            .move({ origin: el, x: 100, y: 0 })
+            .up();
+          const foreFinger = driver.action('pointer')
+            .move({ origin: el, x: 100, y: 0 })
+            .down()
+            .pause(100)
+            .move({ origin: el, x: 50, y: 0 })
+            .up();
+
+          await driver.actions([thumb, foreFinger]);
+        }
+        await doPinch();
+      });
+    });
+    describe('special actions', function () {
+      // TODO: Neither ControlCenterView or ControlCenterView is available on recent iOS.
+      it.skip('should open the control center', async function () {
+        let isStatusBarAvailable = false;
+        try {
+          await driver.$(`${PREDICATE_SEARCH}:type == 'XCUIElementTypeStatusBar'`)
+            .should.eventually.be.rejectedWith(/An element could not be located/);
+        } catch (err) {
+          // if this exists,
+          isStatusBarAvailable = true;
+          await driver.$('~ControlCenterView')
+            .should.eventually.be.rejectedWith(/An element could not be located/);
+        }
+
+        let x, y0, y1;
+        const window = await driver.$(`${PREDICATE_SEARCH}:type == 'XCUIElementTypeApplication'`);
+        const {width, height} = await window.getSize();
+        try {
+          // Try locating the 'Cellular' element (which can be pulled down)
+          const cellularEl = await driver.$('~Cellular');
+          const location = await cellularEl.getLocation();
+          x = location.x;
+          y0 = location.y;
+        } catch (e) {
+          // Otherwise, pull down the middle of the top of the Simulator
+          const isIphoneX = await (async () => {
+            if (UICATALOG_CAPS.deviceName.toLowerCase().includes('iphone x')) {
+              return true;
+            }
+            const { platformVersion, deviceName } = await driver.sessionCapabilities();
+            const translatedDeviceName = translateDeviceName(platformVersion, deviceName).toLowerCase();
+            return _.includes(translatedDeviceName, 'iphone x');
+          })();
+
+          x = width / 2;
+          y0 = isIphoneX
+            ? 15
+            : height - 5;
+        }
+        y1 = height / 2;
+
+        await driver.touchAction([
+          { action: 'press', x, y: y0 },
+          { action: 'moveTo', x, y: y1 },
+        ]);
+
+        // Control Center ought to be visible now
+        if (isStatusBarAvailable) {
+          await driver.$('~ControlCenterView');
+        } else {
+          await driver.$(`${PREDICATE_SEARCH}:type == 'XCUIElementTypeStatusBar'`);
+        }
+      });
+    });
+  });
+  describe('tap with tapWithShortPressDuration cap', function () {
+    // needs a special cap, so has to be in its own session
+    before(async function () {
+      const caps = amendCapabilities(UICATALOG_CAPS, {
+        'appium:tapWithShortPressDuration': 0.01,
```

Comment:
Is `tapWithShortPressDuration` available now? Can we remove this test?

Reply:
no, I don't think it was ever available. Why do we need this capability in general?

Reply:
Okay, I'll remove it later.

Reply:
I deleted this test and submit 2 PRs to cleanup the cap.
https://github.com/appium/appium-xcuitest-driver/pull/1481
https://github.com/appium/java-client/pull/1825